File: shared/src/main/java/xyz/klinker/messenger/shared/receiver/CarReadReceiver.java
package xyz.klinker.messenger.shared.receiver;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import xyz.klinker.messenger.shared.service.NotificationMarkReadService;

public class CarReadReceiver extends BroadcastReceiver {

    @Override
    public void onReceive(Context context, Intent intent) {
        NotificationMarkReadService.handle(intent, context);
    }
}

File: shared/src/main/java/xyz/klinker/messenger/shared/receiver/CarReadReceiver.kt
package xyz.klinker.messenger.shared.receiver

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class CarReadReceiver : BroadcastReceiver() {

    override fun onReceive(context: Context, intent: Intent) {
        xyz.klinker.messenger.shared.receiver.notification_action.NotificationMarkReadReceiver.handle(intent, context)
    }
}
